docs/kratos/self-service/flows/verify-email-account-activation.mdx

```diff
@@ -183,408 +134,141 @@ your Identity Schema:
+   ```
 
-:::
+2. Add the configuration for the verification flow
 
-When the `link` method is enabled, it will be part of the `methods` payload in the Verification Flow:
-
-```shell script
-curl -H "Accept: application/json" -s \
-    'https://playground.projects.oryapis.com/self-service/verification/flows?id=956c0499-17a9-4a99-9602-9282d1632f7b' | \
-      jq -r '.ui.nodes[] | select(.group=="link")'
-
-{
-  "type": "input",
-  "group": "link",
-  "attributes": {
-    "name": "email",
-    "type": "email",
-    "required": true,
-    "disabled": false
-  },
-  "messages": null,
-  "meta": {}
-}
-{
-  "type": "input",
-  "group": "link",
-  "attributes": {
-    "name": "method",
-    "type": "submit",
-    "value": "link",
-    "disabled": false
-  },
-  "messages": null,
-  "meta": {
-    "label": {
-      "id": 1070005,
-      "text": "Submit",
-      "type": "info"
-    }
-  }
-}
-```
+   ```diff
+   selfservice:
+     flows:
+       login:
+   +      after:
+   +        hooks:
+   +        - hook: require_verified_address
+   ```
 
-## Verification form validation
+3. Update the Ory Identities configuration using the file you worked with:
 
-The form payloads are then submitted to Ory Identities which follows up with:
+   ```shell
+   ory update identity-config {project-id} --file updated_config.yaml
+   ```
 
-- An HTTP 303 See Other redirect pointing to the Registration UI for Browser Clients;
-- An `application/json` response for API Clients and Client-Side Browser applications (for example Single Page Apps)e Apps).
+:::tip
 
-### Verification Link via Email
+For more information see the [hooks configuration](../../hooks/01_configure-hooks.mdx) documentation.
 
-To send the verification email, the end-user fills out the form. There might be validation errors such as a malformed email:
+:::
 
-<CodeTabs items={getFlowMethodLinkWithErrors} />
+```mdx-code-block
+  </TabItem>
+</Tabs>
+```
 
-When validation errors happen, browser clients receive a HTTP 303 See Other redirect to the Verification Flow UI, containing the
-Verification Flow ID which includes the error payloads.
+## Choosing the right strategy
 
-For API Clients, the server typically responds with HTTP 400 Bad Request and the Verification Flow in the response payload as
-JSON.
+Ory supports two strategies for verifying your user's addresses.
 
-#### Successful submission
+```mdx-code-block
+<Tabs>
+  <TabItem value="code" label="One-time codes">
+```
 
-On successful submission, an email will be sent to the provided address:
+When using one-time codes to verify a user's address, Ory Identities sends out an email containing a 6-digit code. The user must
+enter the code in a dedicated UI text field to verify their address.
 
-<CodeTabs items={getFlowMethodLinkSuccess} />
+Alternatively, the email also contains a link to the form that pre-fills the code if the user clicks on it. The user must still
+submit the form for Ory Identities to actually mark the address as verified.
```

Comment:
```suggestion
The email also contains a link, which takes the user to the verification form with the code pre-filled in the appropriate UI field. The only thing the user must do to verify their address is to submit the form. 
```

Reply:
done, not sure why `Now, the user only has to submit the form to complete the verification.` isn't good though....